doc/debugger.rst

```diff
@@ -1773,6 +1811,161 @@ target endianness and interpret the bytecode based on that.
 .. note:: This command is somewhat incomplete at the moment and may be modified
    once the best way to do this in the debugger UI has been figured out.
 
+AppRequest request (0x22)
+-------------------------
+
+Format::
+
+    REQ <int: 0x22> [<tval>*] EOM
+    REP [<tval>*] EOM
+
+Example::
+
+    REQ 34 "GameInfo" "GetTitle" EOM
+    REP "Spectacles: Bruce's Story" EOM
+
```

Comment:
Maybe add an error example for:

- Application error (since we added it for this request)
- Duktape error e.g. "AppRequest unsupported by target" (that has the code "unsupported error")